### ISSUE TITLE
importlib.metadata.PackageNotFoundError: No package metadata was found for ocrd_modelfactory

### ISSUE THREAD

**Issue by mikegerber** [OCR-D/core#1262]
With a clean install of ocrd 2.67.1, e.g.:

```
❯ pyenv virtualenv 3.11 ocrd_test
❯ pyenv activate ocrd_test
❯ pip install ocrd
```

Then:

```
❯ ocrd
[...stack trace ... ]
importlib.metadata.PackageNotFoundError: No package metadata was found for ocrd_modelfactory
```

This also affects (all?) other Python processors!

**Comment by mikegerber** [comment]
Downgrading to ocrd==2.67.0 seems to fix this.

**Comment by mikegerber** [comment]
This is on Python 3.11, might be relevant

**Comment by mikegerber** [comment]
Same problem on Python 3.9.

**Comment by mikegerber** [comment]
This is weird because there seem to be no relevant changes between the 2.67.0 and 2.67.1; maybe a broken PyPI build?

**Comment by mikegerber** [comment]
```
❯ pip-wheel-diff ocrd==2.67.0 ocrd==2.67.1
Collecting ocrd==2.67.0
  Using cached ocrd-2.67.0-py3-none-any.whl.metadata (8.8 kB)
Using cached ocrd-2.67.0-py3-none-any.whl (368 kB)
Saved /tmp/tmp0tapv1_i/ocrd-2.67.0-py3-none-any.whl
Collecting ocrd==2.67.1
  Using cached ocrd-2.67.1-py3-none-any.whl.metadata (8.8 kB)
Using cached ocrd-2.67.1-py3-none-any.whl (342 kB)
Saved /tmp/tmp78oa1s_j/ocrd-2.67.1-py3-none-any.whl
diff -r /tmp/tmpt8r5tzbw/v1/ocrd/cli/__init__.py /tmp/tmpt8r5tzbw/v2/ocrd/cli/__init__.py
82c82
< @click.version_option(package_name='ocrd')
---
> @click.version_option(package_name='ocrd_modelfactory')
Only in /tmp/tmpt8r5tzbw/v1/ocrd: ocrd-all-tool.json
diff -r /tmp/tmpt8r5tzbw/v1/ocrd.dist-info/METADATA /tmp/tmpt8r5tzbw/v2/ocrd.dist-info/METADATA
3c3
< Version: 2.67.0
---
> Version: 2.67.1
diff -r /tmp/tmpt8r5tzbw/v1/ocrd.dist-info/WHEEL /tmp/tmpt8r5tzbw/v2/ocrd.dist-info/WHEEL
2c2
< Generator: setuptools (70.3.0)
---
> Generator: setuptools (71.0.1)
diff -r /tmp/tmpt8r5tzbw/v1/ocrd_utils/constants.py /tmp/tmpt8r5tzbw/v2/ocrd_utils/constants.py
21c21
< VERSION = dist_version('ocrd')
---
> VERSION = dist_version('ocrd_modelfactory')
```

**Comment by kba** [comment]
I had a bit of trouble with the upload, but a fixed v2.67.2 has now been released.
